app/controllers/actions/AuthenticateAction.scala

```diff
@@ -92,4 +98,18 @@ private[actions] class AuthenticateAction @Inject()(override val authConnector:
       logger.warn(s"Failed to authorise", e)
       Left(Unauthorized)
   }
+
+  private def track(channel: ChannelType, enrolmentId: Ior[TURN, EORINumber]) = {
+
+    val enrolmentType = enrolmentId.fold(
+      _ => "Legacy",
+      _ => "Modern",
+      (_, modern) => modern.value
+    )
+    val message = s"Auth Successful: $channel:$enrolmentType"
```

Comment:
Could we use something a bit more structure than this? The audit events are JSON objects so that the CIP team can pull apart the data more easily. I think that embedding the data in a static string is a bit unhelpful for that purpose - it would be better to add both pieces of data as properties in the JSON.
We will also need to speak to the CIP team about this and advise that we are adding new audit events to the service. They will want to review.